### PR TITLE
Fix issue #6781,  and some code clean.

### PR DESCRIPTION
**Pull request by yanggangtony**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?
fix some logs error , and code cleans.

Fixes #(issue)
#6781 

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

File: pkg/controller/pod_volume_backup_controller.go [hunk 1]
```diff
@@ -101,8 +101,6 @@ func (r *PodVolumeBackupReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		)
 	}
 
-	log.Info("PodVolumeBackup starting")
```
Comment (qiuming-best) [inline pkg/controller/pod_volume_backup_controller.go:104]: Maybe it's more suitable to  move this log to L116, only "", `velerov1api.PodVolumeBackupPhaseNew` will print this log

Comment (yanggangtony) [reply pkg/controller/pod_volume_backup_controller.go:104]: Thanks.
Make sense to me.
Rebased.

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by qiuming-best — APPROVED** [review]

**Comment by qiuming-best** [comment]
As it was [cherry-picked](https://github.com/vmware-tanzu/velero/pull/6899) to 1.12.1, so I'll remove the `target/1.12.1` label

**Pull request merged**